src/SDK/Language/Node.php

```diff
@@ -164,8 +167,8 @@ public function getFiles(): array
             ],
             [
                 'scope'         => 'default',
-                'destination'   => 'src/inputFile.ts',
-                'template'      => 'node/src/inputFile.ts.twig',
+                'destination'   => 'src/payload.ts',
+                'template'      => 'node/src/Payload.ts.twig',
```

Comment:
```suggestion
                'template'      => 'node/src/payload.ts.twig',
```
use lowercase files names